src/main/java/thePackmaster/SpireAnniversary5Mod.java

```diff
@@ -162,50 +164,87 @@ public void receiveEditCards() {
     @Override
     public void receivePostInitialize() {
         declarePacks();
+        BaseMod.logger.info("Full list of packs: " + allPacks.stream().map(pack -> pack.name).collect(Collectors.toList()));
     }
 
     @Override
     public void receiveEditStrings() {
         BaseMod.loadCustomStringsFile(CardStrings.class, modID + "Resources/localization/" + getLangString() + "/Cardstrings.json");
-
         BaseMod.loadCustomStringsFile(RelicStrings.class, modID + "Resources/localization/" + getLangString() + "/Relicstrings.json");
-
         BaseMod.loadCustomStringsFile(CharacterStrings.class, modID + "Resources/localization/" + getLangString() + "/Charstrings.json");
-
         BaseMod.loadCustomStringsFile(PowerStrings.class, modID + "Resources/localization/" + getLangString() + "/Powerstrings.json");
-
         BaseMod.loadCustomStringsFile(UIStrings.class, modID + "Resources/localization/" + getLangString() + "/UIstrings.json");
-
         BaseMod.loadCustomStringsFile(StanceStrings.class, modID + "Resources/localization/" + getLangString() + "/Stancestrings.json");
-
         BaseMod.loadCustomStringsFile(OrbStrings.class, modID + "Resources/localization/" + getLangString() + "/Orbstrings.json");
+
+        loadPackStrings();
+    }
+
+    // These packs are excluded from loading of pack-specific string files because they consistent entirely of base game cards.
+    // If you're making a pack that also consists of only base game cards, add it to this list.
+    // The name and description of the pack can go in the main UIstrings.json file.
+    private static final List<String> baseGamePacks = Arrays.asList(IroncladPack.class.getName(), SilentPack.class.getName(), DefectPack.class.getName(), WatcherPack.class.getName());
```

Comment:
This is the one central line of code that could result in merge conflicts now, but there are very few packs with only base game cards so I doubt it will be an issue.